pocs/scheduler/constraint.py

```diff
@@ -159,3 +159,23 @@ def get_score(self, time, observer, observation, **kwargs):
 
     def __str__(self):
         return "Moon Avoidance"
+
+class Seqencer(BaseConstraint):
```

Comment:
Seqencer -> Sequencer

But let's maybe name that `AlreadyVisited` just to be really obvious. Sequencer seems to apply to more than one targets whereas a constraint technically only applies to one target at a time.